docs/_specification/1.2-DRAFT/root-data-entity.md

```diff
@@ -128,14 +127,410 @@ be minimally valid.
 
 ## Direct properties of the Root Data Entity
 
-The _Root Data Entity_ MUST have the following properties:
+The _Root Data Entity_ of _Valid RO-Crate Dataset_ 
+Skip to content
+Search Gists
+All gists
+Back to GitHub
+@SamHames
+SamHames/rocrate_relational.py
+Created January 9, 2025 13:31 • Report abuse
+
```

Comment:
This code snippet has been probably pasted by mistake.

Reply:
Fixed - so sorry I was rushing to commit before the meeting yesterday